pkg/csi/nodeutils.go

```diff
@@ -62,17 +60,13 @@ func (ns *nodeServer) createLV(ctx context.Context, req *csi.NodePublishVolumeRe
 	}
 	devicePath := filepath.Join("/dev/", vgName, volumeID)
 	if _, err := ns.osTool.Stat(devicePath); os.IsNotExist(err) {
-		newDev, bdevName, err := ns.createVolume(req.VolumeContext, volumeID, vgName, lvmType)
+		_, _, err := ns.createVolume(req.VolumeContext, volumeID, vgName, lvmType)
```

Comment:
bdev is the component of spdk, so the second return value(bdevName) of createVolume can be removed.

Reply:
current function 'createLV' should follow the same way.